backend/tests/integration/tests/streaming_endpoints/utils.py

```diff
@@ -0,0 +1,24 @@
+from typing import Any
+from typing import Dict
+from typing import List
+
+from tests.integration.common_utils.test_models import StreamedResponse
+
```

Comment:
I think this should be a function in the chatmanager called like "translate_response".

So we can keep all the utilities for handling the api responses in the manager class. Feels a bit odd to have utils in 2 different places.